### DISCUSSION TITLE
trusted.gpg deprecated (update wiki)

### DISCUSSION THREAD

**Discussion by TQQEU** [wireapp/wire-desktop#5712] [Ideas]
Hi,

as there is no way to make a PR to a wiki in Github:
I was informed by my distribution (Kubuntu 22.04), that the usage of trusted.gpg (/etc/apt/trusted.gpg) is deprecated.

To fix the warning I had to do the following steps:
``` bash
sudo apt-key del ABBA007D6E14E2DB5B283C45D599C1AA126762B1
wget -qO- https://wire-app.wire.com/linux/releases.key | sudo tee /etc/apt/trusted.gpg.d/wire.asc
```

That way is advised by the `apt-key`-manpage.

I suggest you to replace this line from [your documentation](https://github.com/wireapp/wire-desktop/wiki/How-to-install-Wire-for-Desktop-on-Linux)

``` bash
wget -q https://wire-app.wire.com/linux/releases.key -O- | sudo apt-key add -
```

by that

``` bash
wget -qO- https://wire-app.wire.com/linux/releases.key | sudo tee /etc/apt/trusted.gpg.d/wire.asc
```

greetings,
Markus

**Comment by TQQEU** [comment]
To make this google-able – This is the text from the warning, I got:
```
W: https://wire-app.wire.com/linux/debian/dists/stable/InRelease: Schlüssel ist im veralteten Schlüsselbund trusted.gpg gespeichert (/etc/apt/trusted.gpg), siehe den Abschnitt MISSBILLIGUNG in apt-key(8) für Details.
```

**Comment by RandomHuman2020** [comment]
I was able to fix this using the recommended method listed in [apt-key(8)'s DEPRECATION section](https://manpages.debian.org/testing/apt/apt-key.8.en.html#DEPRECATION).
Here's an example:
`deb [arch=amd64 signed-by=/etc/apt/keyrings/wire-desktop.key]  https://wire-app.wire.com/linux/debian stable main`
This example does require modification to the download code of the cert as well, and would be:
`wget -q https://wire-app.wire.com/linux/releases.key -O- | sudo tee /etc/apt/keyrings/wire-desktop.key`
I have already made local changes to the wiki, though as mentioned in the original post for this discussion, you cannot make PRs to wiki pages.
I have uploaded my local changes for the wiki to [RandomHuman2020/wiredesktopwiki](https://github.com/RandomHuman2020/wiredesktopwiki) but it'll be up to the developers of Wire Desktop for Linux to decide on if they're gonna pull the changes to the wiki page.
